pioneers-testnet/pioneers-exercise-2.md

```diff
@@ -0,0 +1,170 @@
+# Shelley Stakepool Pioneers Exercise Sheet 2
+
+## Basic Transactions on the Cardano Blockchain
+
+In the first exercise, we set up a Cardano node and ran it.  In this exercise, we will build transactions that are submitted to the Blockchain.  Transactions are the basic mechanism that is used to transfer Ada between stakeholders, register staking pools, stake Ada, and many other things.
+
+### Prerequisites
+
+1. Complete Exercise Sheet 1, and confirm that you have successfully built and run a node.  Also make sure that you have requested some test Ada through the spreadsheet.
+
+2. Read the IOHK Tutorial Documentation and General Documentation on Basic Transactions, Metadata, Addresses, Blocks and Slots at:
+a. https://github.com/input-output-hk/cardano-tutorials/
+b. https://testnet.cardano.org/
+
+3. Checkout the latest version of the Shelley node and CLI from source, and rebuild and reinstall them if they have changed:
+
+```bash
+$ git checkout pioneer
+$ cabal build all
+$ …
+```
+
+### Objectives
+
+In the second exercise, we will make sure that you can:
+
+1. Build simple transactions using the basic transaction mechanism;
+2. Sign transactions and confirm that the transaction is complete;
+3. Submit transactions to the Pioneer Blockchain;
+4. Verify that the transactions have been processed by inspecting the addresses that they have been sent to.
+
+This unlocks many of the things that we will want to do with the Cardano Blockchain in future, including transferring Ada, stake pool registration, and staking.
+
+If you have any questions or encounter any problems, please feel free to use the dedicated Friend & Family Telegram channel.  IOHK staff will be monitoring the channel, and other Pioneers may also be able to help you.
+
+### Exercises
+
+1. Make sure that you do not have an old instance of the node running:
+
+```bash
+$ killall cardano-node
+```
+
+Start a new instance of the node, as you did in Exercise 1:
+
+```bash
+$ cardano-node run --config …
+```
+
+Verify that your new node instance is running:
+
+```bash
+$ ps x | grep cardano-node
+10765 pts/4 R + 1:20  cardano-node …
+$
+```
+
+Your node should be connected to the Pioneer Testnet and verifying the blocks that it receives.
+
+2. 	Verify that you have received some test Ada at the address that you provided to IOHK in Exercise 1, myaddr.
+
+```
+$ cardano-cli shelley query filtered-utxo \
+ 	--address ... \
+ 	--network-magic …
```

Comment:
```suggestion
 	--testnet-magic …
```
This option has been renamed.

Reply:
I am still confused about the magic option

Reply:
It's still `--network-magic`